app/models/manager_refresh/inventory.rb

```diff
@@ -1,61 +1,5 @@
-module ManagerRefresh
-  class Inventory
-    require_nested :Collector
-    require_nested :Parser
-    require_nested :Persister
-
-    attr_accessor :collector, :parsers, :persister
-
-    # Based on the given provider/manager class, this returns correct parser class
-    #
-    # @param klass class of the Provider/Manager
-    # @return [Class] Correct class name of the Parser
-    def self.parser_class_for(klass)
-      provider_module = ManageIQ::Providers::Inflector.provider_module(klass)
-      "#{provider_module}::Inventory::Parser::#{klass.name.demodulize}".safe_constantize
-    rescue ManageIQ::Providers::Inflector::ObjectNotNamespacedError => _err
-      nil
-    end
-
-    # Based on the given provider/manager class, this returns correct persister class
-    #
-    # @param klass class of the Provider/Manager
-    # @return [Class] Correct class name of the persister
-    def self.persister_class_for(klass)
-      provider_module = ManageIQ::Providers::Inflector.provider_module(klass)
-      "#{provider_module}::Inventory::Persister::#{klass.name.demodulize}".safe_constantize
-    rescue ManageIQ::Providers::Inflector::ObjectNotNamespacedError => _err
-      nil
-    end
-
-    # @param persister [ManagerRefresh::Inventory::Persister] A Persister object
-    # @param collector [ManagerRefresh::Inventory::Collector] A Collector object
-    # @param parsers [ManagerRefresh::Inventory::Parser|Array] A Parser object or an array of
-    #   ManagerRefresh::Inventory::Parser objects
-    def initialize(persister, collector, parsers)
-      @collector = collector
-      @persister = persister
-      @parsers   = parsers.kind_of?(Array) ? parsers : [parsers]
-    end
-
-    # Invokes all associated parsers storing parsed data into persister.inventory_collections
-    #
-    # @return [ManagerRefresh::Inventory::Persister] persister object, to allow chaining
-    def parse
-      parsers.each do |parser|
-        parser.collector = collector
-        parser.persister = persister
-        parser.parse
-      end
-
-      persister
-    end
-
-    # Returns all InventoryCollections contained in persister
-    #
-    # @return [Array<ManagerRefresh::InventoryCollection>] List of InventoryCollections objects
-    def inventory_collections
-      parse.inventory_collections
-    end
-  end
+class ManagerRefresh::Inventory < ManageIQ::Providers::Inventory
```

Comment:
I added this as a placeholder so that we can merge this green and move the rest of the provider plugins over, then we can delete this once they're all updated, rather than having to do a mass merge red.